packages/webpack/lib/configs/build.js

```diff
@@ -127,13 +127,8 @@ module.exports = function getConfig(config, name) {
       },
       minimizer: [
         new TerserPlugin({
-          cache: true,
-          parallel: true,
-          sourceMap: true,
+          extractComments: false,
           terserOptions: {
-            compress: {
-              inline: 1, // https://github.com/mishoo/UglifyJS2/issues/2842
```

Comment:
https://github.com/mishoo/UglifyJS2/issues/2842 is still open, are we sure that this is fixed now?

Reply:
The last comment says that it is fixed and I tried the initial test case from the report and it seems to work: https://github.com/mishoo/UglifyJS2/issues/2842#issuecomment-415642909

Reply:
The last comment sais that it is fixed and I tried it with the reproduction test cases from the issue report: https://github.com/mishoo/UglifyJS2/issues/2842#issuecomment-415642909